### ISSUE TITLE
[Bug]: ERROR: python3-venv is not installed, aborting...

### ISSUE THREAD

**Issue by comienzo2093** [brkirch/stable-diffusion-webui#2]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

I ran webui.sh on M1pro, macos 13.3 And I get the following error message:

`ERROR: python3-venv is not installed, aborting...`

automatic1111's webui worked normally with this webui-user.sh file(https://github.com/AUTOMATIC1111/stable-diffusion-webui/files/10554540/webui-user.sh.zip)https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/7453#discussion-4812369).
What could be the problem?

### Steps to reproduce the problem

1. Go to .... 
2. Press ....
3. ...

### What should have happened?

webui should run normally.

### Commit where the problem happens

4b15929

### Command Line Arguments

```Shell
No
```

### List of extensions

No

### Console logs

```Shell
################################################################
Install script for stable-diffusion + Web UI
Tested on Debian 11 (Bullseye)
################################################################

################################################################
Running on hoon user
################################################################

################################################################
Repo already cloned, using it as install directory
################################################################

################################################################
ERROR: python3-venv is not installed, aborting...
################################################################
```

**Comment by comienzo2093** [comment]
```
python3 --version
Python 3.10.10
```

I completely deleted and reinstalled python, but it made no difference.

**Comment by brkirch** [comment]
Are you trying to use the repository directly? This repository is intended to be loaded by the installer [here](https://github.com/brkirch/stable-diffusion-webui/releases) rather than directly.

The installer creates the venv and adds a standalone copy of python (originally created by [portable-python](https://pypi.org/project/portable-python/)) and a standalone copy of git (taken from homebrew and manually configured with `install_name_tool` to use relative paths). Also it adds a modified default configuration and some models required to get Stable Diffusion working.

In other words, you don't need to worry about the venv, the python install, installing command line tools for git or anything else. Run the installer, drop your Stable Diffusion models in `models/Stable-diffusion`, open `Launch Web UI` from the created folder after the installer quits and it should work.

**Comment by comienzo2093** [comment]
> 

thank you. I was stupid It works very well now. I'll test it out soon and leave feedback.
